README.md

```diff
@@ -49,36 +49,29 @@ Docker](#with-docker).
 Install TigerBeetle by grabbing the latest release from
 GitHub.
 
-#### Prebuilt Linux binary
+x86_64 and aarch64 builds are available for macOS and Linux. Only
+x86_64 builds are available for Windows.
 
-```bash
-$ curl -LO https://github.com/tigerbeetledb/tigerbeetle/releases/download/2022-11-16-weekly/tigerbeetle-Linux-x64-2022-11-16-weekly.zip
-$ unzip tigerbeetle-Linux-x64-2022-11-16-weekly.zip
-$ sudo cp tigerbeetle /usr/local/bin/tigerbeetle
-$ tigerbeetle version --verbose | head -n6
-TigerBeetle version experimental
-
-git_commit="b47292aaf2492e6b56a977009b85f7fca6e66775"
-
-zig_version=0.9.1
-mode=Mode.ReleaseSafe
-```
-
-#### Prebuilt macOS binary
+For example:
 
 ```bash
-$ curl -LO https://github.com/tigerbeetledb/tigerbeetle/releases/download/2022-11-16-weekly/tigerbeetle-macOS-x64-2022-11-16-weekly.zip
-$ unzip tigerbeetle-macOS-x64-2022-11-16-weekly.zip
-$ sudo cp tigerbeetle /usr/local/bin/tigerbeetle
+$ curl -LO https://github.com/tigerbeetledb/tigerbeetle/releases/download/2023-03-27-weekly/tigerbeetle-x86_64-linux-2023-03-27-weekly.zip
+$ unzip tigerbeetle-x86_64-linux-2023-03-27-weekly.zip
+$ sudo cp tigerbeetle /usr/local/bin/tigerbeetle # On Windows, add this directory to $env:PATH instead.
```

Comment:
Maybe "add the current director" instead of "add this directory", since "this" could refer to `/usr/local/bin/tigerbeetle`.

Reply:
Updated to be clearer